### PR TITLE
Few fixes / amends in order to cleanup the UI

### PR DESCRIPTION
**Pull request by francescozaia**
- User menu
- "Cos'è docs italia" page
- Log out page
- Homepage Search box
- Language selection dropdown

### CODE REVIEW

**Review by xrmx — APPROVED** [review]

**Comment by francescozaia** [comment]
@yakky ho aggiunto un rapido commit che completa la [scheda su Trello](https://trello.com/c/4HLob6un/68-correzioni-tema), quando riesci puoi darci un occhio e mergiare, grazie!

File: readthedocs/templates/docsitalia/overrides/projects/project_dashboard_base.html [hunk 1]
```diff
@@ -151,6 +151,7 @@ <h3>{% trans "Connect your Accounts" %}</h3>
   {% block dashboard-onboard-extra %}
   {% endblock %}
 
+  {% comment %}
```
Comment (xrmx) [inline readthedocs/templates/docsitalia/overrides/projects/project_dashboard_base.html]: please don't :) remove don't comment

File: readthedocs/templates/docsitalia/overrides/projects/project_dashboard.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{% extends 'projects/project_dashboard_base.html' %}
+
+{% block dashboard-onboard-extra %}
```
Comment (xrmx) [inline readthedocs/templates/docsitalia/overrides/projects/project_dashboard.html]: The right fix is the following:

```
{% block dashboard-onboard-extra %}{% endblock %}
```

**Review by xrmx — CHANGES_REQUESTED** [review]

File: readthedocs/templates/docsitalia/includes/header.html [hunk 3]
```diff
@@ -43,15 +43,39 @@
...
+        </div>
+        <div class="menu-wrapper">
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <a class="nav-link" href="/che-cos-e-docs-italia/" aria-label="Che cos'è Docs Italia" title="Che cos'è Docs Italia"><b>Che cos'è?</b></a>
+            </li>
+            {% if request.user.is_authenticated %}
+            {% include 'docsitalia/includes/user_menu.html' %}
+            {% else %}
+            {% comment %}
```
Comment (yakky) [inline readthedocs/templates/docsitalia/includes/header.html]: Any reason not to remove this?

File: readthedocs/templates/docsitalia/includes/user_menu.html [hunk 4]
```diff
@@ -1,20 +1,25 @@
...
 {% load gravatar %}
-<div class="menu menu-user" id="subnav-user-menu">
-  <div class="menu-button">
-    <a href="{% url 'projects_dashboard' %}">
-      {% gravatar request.user.email 24 %}
-      {{ request.user }}</a>
-    <button class="menu-drop">▼</button>
+<li class="nav-item dropdown">
+  <a class="nav-link dropdown-toggle" href="#" data-toggle="dropdown" aria-expanded="false"><span>
+    {% if user.get_full_name %}
```
Comment (yakky) [inline readthedocs/templates/docsitalia/includes/user_menu.html]: I am not sure we have `user` in the context. Maybe `request.user` ?

File: readthedocs/templates/docsitalia/overrides/core/header.html [hunk 5]
```diff
@@ -19,7 +19,7 @@ <h1>
     <div class="menu menu-user" id="subnav-user-menu">
       <div class="menu-button">
         <a href="{% url 'projects_dashboard' %}">
-          {% gravatar request.user.email 24 %}
+          {% comment %}{% gravatar request.user.email 24 %}% endcomment %}
```
Comment (yakky) [inline readthedocs/templates/docsitalia/overrides/core/header.html]: Maybe we should just remove this?

File: readthedocs/templates/docsitalia/overrides/projects/project_dashboard_base.html (hunk 1 above)
Comment (francescozaia) [reply readthedocs/templates/docsitalia/overrides/projects/project_dashboard_base.html]: 👍

File: readthedocs/templates/docsitalia/overrides/projects/project_dashboard_base.html [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+      <button type="submit" class="btn btn-outline-primary">{% trans "Connect your Accounts" %}aaaaa</button>
+    </form>
+  </div>
+  {% endif %}
+  <!-- END social account onboard -->
+
+  {% block dashboard-onboard-extra %}
+  {% endblock %}
+
+  {% comment %}
```
Comment (yakky) [inline readthedocs/templates/docsitalia/overrides/projects/project_dashboard_base.html]: Maybe we should just remove this?

File: readthedocs/templates/docsitalia/overrides/projects/project_dashboard.html (hunk 2 above)
Comment (francescozaia) [reply readthedocs/templates/docsitalia/overrides/projects/project_dashboard.html]: grazie, sistemo subito

**Review by yakky — CHANGES_REQUESTED** [review]
Can you check the minor changes reported?

**Review by yakky — APPROVED** [review]

**Comment by yakky** [comment]
LGTM. Waiting for @xrmx review to merge

**Review by xrmx — APPROVED** [review]

**Pull request merged**